src/components/ChartComponent/ChartComponentView.jsx

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable react/forbid-prop-types */
+import React from 'react';
+import PropTypes from 'prop-types';
+import Chart from '../Chart/Chart';
+import TimeSwitcher from '../Switcher/TimeSwitcher';
+import CasesSwitcher from '../Switcher/CasesSwitcher';
+import StageSwitcher from '../Switcher/StageSwitcher';
+import style from './ChartComponent.module.css';
+
+export default function ChartComponentView({
+  switchersState,
+  onSwitcherChange,
+  startData,
+  chartData,
+  country,
+}) {
+  return (
+    <div className={style.chart}>
+      <div className={style['chart__switcher-top']}>
+        <TimeSwitcher
+          groupName="chartTime"
+          switchersState={switchersState}
+          onSwitcherChange={onSwitcherChange}
+        />
+      </div>
+      <div className={style['chart__switcher-top']}>
+        <CasesSwitcher
+          groupName="chartCases"
+          switchersState={switchersState}
+          onSwitcherChange={onSwitcherChange}
+        />
+      </div>
+      <div className={style.chart__graph}>
+        {/* <canvas className={style['chart__graph-item']} ref={(el) => { chartRef = el; }} /> */}
```

Comment:
Delete this comment

Reply:
https://github.com/SorokinTim/covid-dashboard-rss/pull/20/commits/f603d5b77ad007455362acc6faa76fff2b99de56